Purchases/Public/Purchases.swift

```diff
@@ -1092,7 +1094,7 @@ public extension Purchases {
      *
      * - Note: Use this initializer if your app does not have an account system.
      * `Purchases` will generate a unique identifier for the current device and persist it to `NSUserDefaults`.
-     * This also affects the behavior of ``Purchases/restoreTransactions(completionBlock:)``.
+     * This also affects the behavior of ``Purchases/restoreTransactions(completion:)``.
```

Comment:
I believe this would be `maybeCompletion`

Reply:
Updated all `public` completion blocks to using just `completion` instead of `maybeCompletion`